resources/validation.txt

```diff
@@ -374,6 +375,7 @@ column-rule-color: currentColor;
 column-rule-style: none;
 column-rule-width: 3px;
 column-fill: auto;
+outline-offset: 0;

```

Comment:
This line is not necessary since `outline-offset` does not take part in any shorthand property.